code/__DEFINES/blood.dm

```diff
@@ -1,5 +1,5 @@
 /// Checks if an object is covered in blood
-#define HAS_BLOOD_DNA(thing) (length(thing.GetComponent(/datum/component/forensics)?.blood_DNA))
+#define HAS_BLOOD_DNA(thing) (length(thing.forensics?.blood_DNA))
```

Comment:
Now that I think about it, why does this macro exist when the helper procs (`blood_DNA_length` and `return_blood_DNA`) exist?
or why do the helper procs exist when they could be macros?
🤔